src/fe-gtk/fe-gtk.c

```diff
@@ -319,10 +319,11 @@ fe_init (void)
 	palette_load ();
 	key_init ();
 	pixmaps_init ();
-
+/*
```

Comment:
More information on whats failing here would be nice.

Reply:
It's more regarding to the error message on the top. For some reason it's causing an exception where the image is sized zero. Still kinda investigating this. But for now it's more of a temp fix.